src/relay/backend/aot_executor_codegen.cc

```diff
@@ -744,13 +744,11 @@ class AOTExecutorCodegen : public MixedModeVisitor {
   }
 
   /*!
-   * brief Calculate workspace sizes for PrimFuncs in the IRModule
+   * \brief Calculate workspace sizes for PrimFuncs in the IRModule
    */
   Map<String, FunctionInfo> CalculateWorkspaceSizes(
       const IRModule& lowered_mod, const Map<String, FunctionInfo>& function_metadata) {
-    Executor executor_config = lowered_mod->GetAttr<Executor>(tvm::attr::kExecutor).value();
-    Integer workspace_byte_alignment =
-        executor_config->GetAttr<Integer>("workspace-byte-alignment").value_or(16);
+    Integer workspace_byte_alignment = GetModuleWorkspaceAlignment(lowered_mod);
```

Comment:
nit: make the name agree with the property, here and below

Reply:
GetModuleWorkspaceAlignment -> GetModuleWorkspaceByteAlignment
GetModuleConstantAlignment -> GetModuleConstantByteAlignment